pkg/cache/tas_cache_test.go

```diff
@@ -1072,3 +1072,67 @@ func TestFindTopologyAssignment(t *testing.T) {
 		})
 	}
 }
+
+func TestLexSortDomains(t *testing.T) {
```

Comment:
I would prefer to test exposed functions, and I believe this function could be made generic. 
I think a good place is `pkg/util/cmp/cmp.go` and name it as `CompareStringSlices()`. I could also make it generic, but can leave it when needed.

Reply:
Done